src/level/LevelDirector.js

```diff
@@ -111,7 +111,7 @@
                         );
                     } else {
                         obj = me.pool.pull(
-                            "me.Entity",
+                            me.pool.exists(settings.className) ? settings.className : "me.Entity",
```

Comment:
that could work too, I just took the safe route to prevent any possible exceptions